src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.php

```diff
@@ -127,6 +127,7 @@
             ->tag('serializer.normalizer', ['priority' => -1000])
 
         ->alias(ObjectNormalizer::class, 'serializer.normalizer.object')
+            ->deprecate('symfony/serializer', '6.2', 'The "%alias_id%" service alias is deprecated, use "'.NormalizerInterface::class.'" instead.')
```

Comment:
isn't implementing NormalizerAwareInterface the right replacement in some cases (like the one described in the docs) ?